network/vcn/securitylists.tf

```diff
@@ -11,6 +11,28 @@ resource "oci_core_security_list" "EtcdSubnet" {
   ]
 
   ingress_security_rules = [
+    {
+      "icmp-options": {
+        "code": 4,    
+        "type": 3
+      },    
+      "is-stateless": false,    
+      "protocol": "1",    
+      "source": "${var.external_icmp_ingress}",    
+      "tcp-options": null,    
+      "udp-options": null
+    },
+    {
+      "icmp-options": {
+        "code": 4,    
+        "type": 3
+      },    
+      "is-stateless": false,    
+      "protocol": "1",    
+      "source": "${var.internal_icmp_ingress}",    
```

Comment:
You need to pass this variable in through `k8s-oci.tf` to the vcn module so that the user can set it i.e. 

```
module "vcn" {
  source                                  = "./network/vcn"
...
  external_icmp_ingress                   = "${var.external_icmp_ingress}"
```